src/content/public goods funding.md

```diff
@@ -0,0 +1,287 @@
+---
+title: Public goods funding 
+description: An overview of public goods funding solutions on Ethereum
+lang: en
+template: use-cases
+emoji: 🌎
+sidebar: true
+sidebarDepth: 2
+summaryPoint1: Public goods are nonexcludable and nonrivalrous and benefit everyone
+summaryPoint2: Traditional models of funding public goods are inefficient and encourage coordination failures
+summaryPoint3: Public goods markets on Ethereum make it easier to fund and develop critical digital public goods
+---
+ 
+Public goods are commodities and services provided to all members of a society (without the possibility of direct profit) usually by government or private organizations. While the creation of public goods is critical to societal progress and wellbeing, traditional models of coordinating to fund public goods suffer problems that limit their efficiency and effectiveness. This makes it difficult to build useful public goods and results in worse outcomes that affect everyone. 
+
+But there might be a solution to the problems of funding public goods: blockchain technology. By applying cryptoeconomic incentives and mechanism designs, new blockchain-based projects are fixing the broken model of traditional public goods funding and solving coordination failures that plague modern markets. 
```

Comment:
Overall, I like this intro. I personally feel like it might benefit from a concrete example or two (e.g. "public goods are things like public parks and clean air..."). What do you think?